autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -46,6 +46,7 @@ $AqtinstallVersion = "2.0.5"
 $JackVersion = "1.9.17"
 $Msvc32Version = "win32_msvc2019"
 $Msvc64Version = "win64_msvc2019_64"
+$JomVersion = "1.1.2"
```

Comment:
Do we have a documented process in place to review pinned versions from time to time?

(Don't I love coming up with unconnected issues...)

Reply:
> Do we have a documented process in place to review pinned versions from time to time?

Not yet, but there's at least *one* reminder: #2346